### DISCUSSION TITLE
How to disable keybinding?

### DISCUSSION THREAD

**Discussion by azzamsa** [sxyazi/yazi#629] [1. Q&A] [answered]
Hi, 👋

I know how to replace/override a key. But how to remove it completely?

- To replace/override.

```toml
prepend_keymap = [
    { on = ["n"], exec = "arrow -1", desc = "Move cursor up" },
    { on = ["e"], exec = "arrow 1", desc = "Move cursor down" },
]
```

- How to disable it?

```toml
prepend_keymap = [
    { on = ["<S-Up>"], exec = "false(?)", desc = "Move cursor up 5 lines" },
    { on = ["<S-Down>"], exec = "false(?)", desc = "Move cursor down 5 lines" },
]
```

Thank you for Yazi! 🍦

**Comment by sxyazi** [comment, marked as answer]
You can bind any commands that do not exist to them, and Yazi will silently ignore them, like:

```toml
prepend_keymap = [
    { on = ["<S-Up>"], exec = "whatever", desc = "Move cursor up 5 lines" },
    { on = ["<S-Down>"], exec = "youwant", desc = "Move cursor down 5 lines" },
]
```

**Reply by azzamsa** [reply]
So, Yazi doesn't check whether the "internal" command exists? What about the typos?
Do you plan to have a better approach to this in the future?

**Reply by sxyazi** [reply]
Yeah if it's a typo, just ignore it

The better approach might be to add a real-existing "noop" command that doesn't do anything, but I don't think it's necessary

**Reply by sxyazi** [reply]
btw you can also completely redefine your keys through ‘keymap’

**Reply by azzamsa** [reply]
> The better approach might be to add a real-existing "noop" command that doesn't do anything, but I don't think it's necessary

Got it.
Thank you so much for Yazi! :ice_cream:

**Reply by sxyazi** [reply]
> The better approach might be to add a real-existing "noop" command that doesn't do anything, but I don't think it's necessary

Added in https://github.com/sxyazi/yazi/pull/1882 since another user also requested this feature
